types/pinyin/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for pinyin 2.8.3
+// Project: https://github.com/hotoo/pinyin
+// Definitions by: AnJun Wang <https://github.com/wanganjun>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = pinyin;
```

Comment:
It seems this packge does not have built-in support for UMD.
https://github.com/hotoo/pinyin/blob/master/examples/index.md
https://github.com/hotoo/pinyin/blob/master/lib/web-pinyin.js